lib/msgraph/paginated.go

```diff
@@ -29,10 +29,17 @@ import (
 	"github.com/gravitational/trace"
 )
 
+const expandParameter = "$expand"
+const expandMemberOf = "memberOf"
+
+type IterateOptions struct {
```

Comment:
@zmb3 like this?

Reply:
Yes, exactly. Now the details of the HTTP API are kept internal to the package :+1: